### DISCUSSION TITLE
How to find OBS Studio recording where the recording was paused and resumed?

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [obsproject/obs-studio#6860] [General]
I would like to find where the video was paused and resumed during the entire recording.

I have a screen recording by OBS studio which is over 7 hours and it has many pause and resume. So I want to find the times / frames / location of this pauses and resumes.

The file type is MKV and the OBS studio is the latest version

I also have asked this question on SO so you can also answer there too

https://stackoverflow.com/questions/73171045/how-to-find-obs-studio-recording-where-the-recording-was-paused-and-resumed-prog

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
